Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -303,42 +305,78 @@ namespace AzToolsFramework::ViewportUi::Internal
         return false;
     }
 
+    QMargins ViewportUiDisplay::CalculateViewportElementMargins() const
+    {
+        if (m_imGuiActive)
+        {
+            if (GetViewportBorderVisible())
```

Comment:
Suggestion: could use the more compact form of `return GetViewportBorderVisible() ? ViewportUiOverlayImGuiBorderMargin : ViewportUiOverlayImGuiMargin;`

Reply:
nit/suggestion: We might also want to rename this to `DetermineViewportElementMargins` as we aren't really performing an expensive calculation here... Maybe just `ViewportElementMargins` would also suffice? 🤔

Reply:
> Suggestion: could use the more compact form of `return GetViewportBorderVisible() ? ViewportUiOverlayImGuiBorderMargin : ViewportUiOverlayImGuiMargin;`

This made it much better, thank you :)